src/translator/output_printer.cpp

```diff
@@ -26,4 +28,12 @@ std::string OutputPrinter::getAlignment(const Ptr<Hypothesis>& hyp) {
   }
 }
 
+std::string OutputPrinter::getWordScores(const Ptr<Hypothesis>& hyp) {
```

Comment:
Comment what these are.

Reply:
I added a more detailed comment to the function generating the scores